src/ewatercycle/models/lisflood.py

```diff
@@ -306,6 +306,11 @@ def parameters(self) -> Iterable[Tuple[str, Any]]:
             ("end_time", self._end.strftime("%Y-%m-%dT%H:%M:%SZ")),
         ]
 
+    def finalize(self) -> None:
+        """Perform tear-down tasks for the model."""
+        # Finalize function of bmi class of lisflood is kaput, so not calling it
```

Comment:
Maybe add a warning here? Or open new issue to remind us to fix it properly?

Reply:
Created https://github.com/eWaterCycle/lisflood-code/issues/1